client/src/components/SectionTable/SectionTableBody.js

```diff
@@ -67,11 +57,6 @@ const styles = (theme) => ({
     full: {
         color: '#e53935',
     },
-    sectionCode: {
-        '&:hover': {
-            cursor: 'pointer',
-        },
-    },
```

Comment:
Duplicate key for `sectionCode`. Added this property to the already existing one above.